importScripts/sql/testImports/itemData.sql

```diff
@@ -71,5 +70,5 @@ SELECT
 FROM
     ObjectsExport
 WHERE
-    PrimaryKey_Object_Table IN(3,6,404,675,679,8,23,25,29,30,47,48,4138,1584,1621,1622,755,1841,3977,4406,3767,4324,4496,5773,679,750,17084,3058,12836,13671,27992,11114,14390,11423,14712,11457,12626,26488,28319,8789,1621,1622)
+    PrimaryKey_Object_Table IN(3,6,404,675,679,8,23,25,47,4138,1584,3977,4406,3767,4324,4496,3058,12836,17084,13671,27992,11114,14390,11423,14712,11457,12626,26488,8789,1621,1622,29,30,48,750,755,1841,5773,28319)
```

Comment:
Thanks for letting me know - would have not really know what to look for otherwise :-)